classes/Mail.php

```diff
@@ -347,22 +348,25 @@ function ($carry, $item) {
                     return false;
                 }
 
-                $connection = \Swift_SmtpTransport::newInstance(
+                $connection = (new Swift_SmtpTransport(
                     $configuration['PS_MAIL_SERVER'],
                     $configuration['PS_MAIL_SMTP_PORT'],
                     $configuration['PS_MAIL_SMTP_ENCRYPTION']
-                )
+                ))
                     ->setUsername($configuration['PS_MAIL_USER'])
                     ->setPassword($configuration['PS_MAIL_PASSWD']);
             } else {
-                $connection = \Swift_MailTransport::newInstance();
+                // mail() support was removed from SwiftMailer for security reasons
+                // previously => $connection = \Swift_MailTransport::newInstance();
+                // @see https://github.com/swiftmailer/swiftmailer/issues/866
+                $connection = false;
```

Comment:
@PierreRambaud That sounds that mail() will not be supported with 1.7.7. May be to indicate somewhere.

Reply:
Woot, didn't check this part :scream: 
We must found a solution because of postfix server :/

Reply:
I add the alternative of Sendmail transport and fix the sendMailTest feature